tests/conftest_extras_w_cci.py

```diff
@@ -0,0 +1,69 @@
+from unittest.mock import patch
+
+import pytest
+from pytest import fixture
+
+from cumulusci.tests.pytest_plugins.pytest_sf_vcr import (
+    vcr_config as cci_vcr_config,
+    salesforce_vcr,
+)
+
+from cumulusci.tests.util import DummyOrgConfig, DummyKeychain
+from cumulusci.core.exceptions import ServiceNotConfigured
+
+DUMMY_ORGNAME = "pytest_sf_orgconnect_dummy_orgconfig"
+
+
+@pytest.fixture(scope="session")
+def fallback_org_config():
+    def fallback_org_config():
+        return DummyOrgConfig(
+            {
+                "instance_url": "https://orgname.my.salesforce.com",
+                "access_token": "pytest_sf_orgconnect_abc123",
+                "id": "ORGID/ORGID",
+            },
+            DUMMY_ORGNAME,
+            keychain=DummyKeychain(),
+        )
+
+    original_get_org = original_load_keychain = None
+
+    def get_org(self, name: str):
```

Comment:
Is there a particular advantage to defining these as inner functions vs. defining them above this current fixture?

Reply:
They need access to `original_get_org` and `original_load_keychain` so this gives them access without making those two variables global.